qa.planx-pla.net/manifest.json

```diff
@@ -7,21 +7,21 @@
     "autodeploy": "yes"
   },
   "versions": {
-    "arborist": "quay.io/cdis/arborist:2020.03",
+    "arborist": "quay.io/cdis/arborist:2.5.0",
     "aws-es-proxy": "abutaha/aws-es-proxy:0.8",
-    "fence": "quay.io/cdis/fence:2020.03",
+    "fence": "quay.io/cdis/fence:2020.10",
     "fluentd": "fluent/fluentd-kubernetes-daemonset:v1.10.2-debian-cloudwatch-1.0",
-    "indexd": "quay.io/cdis/indexd:2020.03",
-    "jenkins": "quay.io/cdis/jenkins:master",
+    "indexd": "quay.io/cdis/indexd:2020.10",
+    "jenkins": "quay.io/cdis/jenkins:2020.10",
```

Comment:
(https://quay.io/repository/cdis/jenkins?tab=tags) looks like there is no 2020.10 image for jenkins in Quay

Reply:
That component / version is not actually being used.